Conduit/Conduit/Controllers/ArticleController.cs

```diff
@@ -78,12 +77,12 @@ public async Task<ActionResult<List<Article>>> GetAllArticle(int PageNumber)
                 var ArticlesNyumberInPage = 3f;
                 var pageCount = Math.Ceiling(_context.Articles.Count() / ArticlesNyumberInPage);
 
-                var articles = await _ArticlesRepositry.GetAllArticlePaginated(PageNumber, ArticlesNyumberInPage);
```

Comment:
hmmm,  I see the changes were merged before in main right?, it's not clear changes in PR